hexrdgui/llnl_import_tool_dialog.py

```diff
@@ -1,18 +1,22 @@
+import glob
 import os
+import re
+from hexrdgui.median_filter_dialog import MedianFilterDialog
```

Comment:
Can you put this with the other `hexrdgui` imports below? It might also be nice to sort all of the imports :slightly_smiling_face: